### PR TITLE
[CosmosDB] Fix references in privateEndpointConnection.json

### PR DESCRIPTION
**Pull request by soeom**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateEndpointConnection.json [hunk 1]
```diff
@@ -46,16 +46,16 @@
         },
         "parameters": [
           {
-            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
```
Comment (anthony-c-martin) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateEndpointConnection.json:49]: Please don't reference files in a different API version. This makes it extremely difficult to untangle dependencies.

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]
See comment

**Comment by soeom** [comment]
@weidongxu-microsoft Please help merge the PR.

**Comment by weidongxu-microsoft** [comment]
@soeom

Please review the CI error on BreakingChange first.

Also Martin still have the "change requested" here.

**Comment by soeom** [comment]
@weidongxu-microsoft 
@anthony-c-martin signed off the pull request by replacing the tag to ARMSignedOff. He also confirmed that he signed off through teams. 
There is no ci failures other than breaking change. This is unavoidable because the purpose of this PR is to consolidate the reference to common parameters. 
Please help merge the PR.

**Comment by weidongxu-microsoft** [comment]
@soeom 

OK. The BreakingChange seems to be a false positive.

Martin had to change "change requested" to "approve", else technically I cannot press merge at all (github forbid it).

**Comment by weidongxu-microsoft** [comment]
@akning-ms 

This BreakingChange CI seems to be a false positive. It reports "removal of client parameter" but none of the change remove any client parameter. It just replaces them.

```
    id: '1001',
    code: 'NoVersionChange',
    message: 'The versions have not changed.',
    old: {
      ref: 'file:///home/vsts/work/1/c93b354fd9c14905bb574a8834c4d69b/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json#',
      path: '',
      location: 'file:///home/vsts/work/1/c93b354fd9c14905bb574a8834c4d69b/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json:1:1'
    },
    new: {
      ref: 'file:///tmp/resolved/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json#',
      path: '',
      location: 'file:///tmp/resolved/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json:1:1'
    },
    type: 'Info',
    docUrl: 'https://github.com/Azure/openapi-diff/tree/master/docs/rules/1001.md',
    mode: 'Update'
  },
  {
    id: '1007',
    code: 'RemovedClientParameter',
    message: "The new version is missing a client parameter that was found in the old version. Was 'apiVersionParameter' removed or renamed?",
    old: {
      ref: 'file:///home/vsts/work/1/c93b354fd9c14905bb574a8834c4d69b/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json#/parameters',
      path: 'parameters',
      location: 'file:///home/vsts/work/1/c93b354fd9c14905bb574a8834c4d69b/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json:174:3'
    },
    new: {
      ref: 'file:///tmp/resolved/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json#/parameters',
      path: 'parameters',
      location: 'file:///tmp/resolved/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json:255:3'
    },
    type: 'Error',
    docUrl: 'https://github.com/Azure/openapi-diff/tree/master/docs/rules/1007.md',
    mode: 'Removal'
  },
```

**Review by anthony-c-martin — APPROVED** [review]

**Pull request merged**